kuka_lbr_iiwa_support/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ find_package(catkin REQUIRED)
 
 catkin_package()
 
-foreach(dir config launch meshes/lbr_iiwa_14_r820/collision meshes/lbr_iiwa_14_r820/visual urdf) 
+foreach(dir config launch meshes urdf) 
```

Comment:
I noticed that you cleaned this up recently in the abb_experimental repository, do you want to fixes this now?
> install(DIRECTORY config launch meshes urdf
    DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})